src/server/gamebananaMirror/yamlHandlers/utils/getFileSystemPath.ts

```diff
@@ -0,0 +1,16 @@
+import path from "path";
+
+
+
+
+/** Takes a file name and gives an absolute file path for a file with that name in `projectRoot/cache/` */
+export const getFileSystemPath = (fileName: string) => {
+    if (!fileName.endsWith(".json")) throw "The file name must end with '.json'.";
+
+
+    const filePath = path.resolve("~/../cache", fileName);
```

Comment:
Does `"cache"` instead of `"~/../cache"` work?

Reply:
Yep